src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -28,103 +44,16 @@
     <DefineConstants>$(DefineConstants);LACKS_CALLERARGUMENTEXPRESSION</DefineConstants>
   </PropertyGroup>
 
-  <!-- .NET Standard 2.0 -->
+  <!-- Pre-.NET Standard 2.1 -->
```

Comment:
Kind of a bonus change. To be honest, I thought it was going to be more useful than it turned out to be. I don't mind the consolidation below though. Still, if you prefer, I can revert!